statsmodels/tsa/arima/estimators/tests/test_durbin_levinson.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+
+import pytest
+from numpy.testing import assert_allclose, assert_raises
+
+from statsmodels.tsa.innovations.arma_innovations import arma_innovations
+from statsmodels.tsa.arima.datasets.brockwell_davis_2002 import dowj, lake
+from statsmodels.tsa.arima.estimators.durbin_levinson import durbin_levinson
+
+@pytest.mark.low_precision('Test against Example 5.1.1 in Brockwell and Davis'
```

Comment:
i expect strict linting would want another newline before this.  are these files not in the strict list?

Reply:
Thanks - I think they are missing it because of the decorator (i.e. when I run `pycodestyle` manually it doesn't pick up anything). I added the directory to the linter list - is that enough?